packages/react-form-state/src/FormState.ts

```diff
@@ -0,0 +1,351 @@
+import * as React from 'react';
+import isEqual from 'lodash/isEqual';
+import isArray from 'lodash/isArray';
+import memoize from 'lodash/memoize';
+
+import {mapObject} from './utilities';
+import {FieldDescriptors, ClientError, FieldState} from './types';
+import {List, Nested} from './components';
+
+const ERROR_SEPARATOR = ', ';
+
+interface GraphQLError {
```

Comment:
Can we name it a little more generically?

Reply:
RemoteError?